app/services/page_updater.rb

```diff
@@ -82,7 +82,11 @@ def update_page
     return unless @params[:page]
     plugins_before = @page.plugins
     @page.assign_attributes(@params[:page])
-    @page.save
+
+    if @page.save
+      QueueManager.push(@page, job_type: :update)
```

Comment:
hahah nice. only tricky bit is tags but you're a sharp lad I'm sure you can find a low-touch way to see if that changed.